src/pybind/mgr/dashboard/frontend/src/app/shared/models/topic.model.ts

```diff
@@ -0,0 +1,43 @@
+interface Destination {
+  push_endpoint: string;
+  push_endpoint_args: string;
+  push_endpoint_topic: string;
+  stored_secret: string;
+  persistent: boolean;
+  persistent_queue: string;
+  time_to_live: number;
+  max_retries: number;
+  retry_sleep_duration: number;
```

Comment:
Camel case

```suggestion
  pushEndpoint: string;
  pushEndpointArgs: string;
  pushEndpointTopic: string;
 ...
```

Reply:
this is being used for typing the data coming from the backend that is snaked_case so what I said does not make sense, marking it as resolved